doc/pages/administrator/064-setup_astartedefaultingress.md

```diff
@@ -0,0 +1,152 @@
+
+Most information needed for exposing your Ingress have already been given in your main Astarte
+resource. If your Kubernetes installation supports LoadBalancer ingresses (most managed ones do),
+you should be able to get away with the most standard CR:
+
+```yaml
+apiVersion: ingress.astarte-platform.org/v1alpha1
+kind: AstarteDefaultIngress
+metadata:
+  name: adi
+  namespace: astarte
+spec:
+  ### Astarte Default Ingress CRD
+  astarte: astarte
+  tlsSecret: <your-astarte-tls-cert>
+  api:
+    exposeHousekeeping: true
+  dashboard:
+    deploy: true
+    ssl: true
+    host: <your-astarte-dashboard-host>
+  broker:
+    deploy: true
+    serviceType: LoadBalancer
+```
+
+There's one very important thing to be noted: the `astarte` field must reference the name of an
+existing Astarte installation in the same namespace, and the Ingress will be configured and attached
+to that instance.
+
+## What happens after installing the AstarteDefaultIngress resource?
+
+When the AstarteDefaultIngress resource is created, the Astarte Operator ensures that the following
+resources are created according to your configuration:
+- an NGINX ingress which is devoted to routing requests to the Astarte APIs and to the Astarte
+  Dashboard,
+- a service of kind LoadBalancer which exposes the Astarte broker to the outer world.
+
+The following commands will help you in the task of retrieving the external IPs assigned to both the
+ingress and the broker service. Assuming that your Astarte instance is named `astarte` and is
+deployed within the `astarte` namespace, simply run:
+
+```bash
+$ # retrieve information about the ingress
+$ kubectl get ingress -n astarte
+```
+
+and
+```bash
+$ # retrieve information about the broker service
+$ kubectl get service -n astarte astarte-broker-service -o wide
+```
+
+## SSL and Certificates
+
+Astarte heavily requires SSL in a number of interactions, even though this can be bypassed with
+`ssl: false`. If you do not have any SSL certificates for your domains, you can leverage
+cert-manager capabilities. Simply follow the instructions outlined
+[here](050-handling_certificates.html) to learn how to handle your certificates.
+
+## How to support automatic certificate renewal for HTTP challenges?
+
+When your certificate is issued after the solution of an HTTP challenge, to ensure the renewal of
+the certificate itself you must ensure that the NGINX ingress and the broker service are exposed on
+the same external IP.
+
+Given that the ingress external IP is obtained after the deployment of the NGINX ingress controller,
+all you have to do is ensuring that the broker service is exposed on the ingress IP. Thus, set the
+`loadBalancerIP` field in your AstarteDefaultIngress resource:
+
+```bash
+apiVersion: ingress.astarte-platform.org/v1alpha1
+kind: AstarteDefaultIngress
+...
+spec:
+  ...
+  broker:
+    deploy: true
+    serviceType: LoadBalancer
+    loadBalancerIP: <same-IP-of-your-ingress>
```

Comment:
I'm afraid that describing the `loadBalancerIP` after the copy-paste ADI CR above may lead people into skipping this important step.
We can still describe the use of that parameter in its own section, but i would add it to the standard CR so that people reading this doc for the first time cannot skip it

Reply:
Added the field with comments. Will it be enough? :cold_sweat: